apps/montreal_elixir/lib/montreal_elixir/social_feeds/meetup_api_client.ex

```diff
@@ -0,0 +1,68 @@
+if Mix.env == :test || Mix.env == :travis do
+  Code.require_file("../../../test/montreal_elixir/social_feeds/api_clients/meetup_api_http_client.exs", __DIR__)
+end
+
+defmodule MontrealElixir.SocialFeeds.MeetupApiClient do
```

Comment:
Since `SocialFeeds` could eventually be used some something else (like videos), maybe this module be namespaced: `MontrealElixir.SocialFeeds.Meetup.ApiClient`, and the file in folder `meetup`.